contract/contracts/forward_name_resolver/reward_token.ral

```diff
@@ -0,0 +1,44 @@
+import "std/fungible_token_interface"
+
+Contract RewardToken(
+    parentId: ByteVec,
+    symbol: ByteVec,
+    name: ByteVec,
+    decimals: U256,
+    mut totalSupply: U256
+) extends Constants() implements IFungibleToken {
+
+    @using(assetsInContract = true, checkExternalCaller = false, updateFields = true)
```

Comment:
[Nit]: No need to add the `checkExternalCaller = false` since the function will check the caller.

Reply:
fixed